1.2/test/run

```diff
@@ -157,9 +158,9 @@ test_application() {
 
   test_scl_usage "elixir --version" "OTP 18"
   check_result $?
-  test_scl_usage "elixir --version" "Elixir 1.3.1"
+  test_scl_usage "elixir --version" "Elixir 1.2.5"
```

Comment:
Since I started with the newest version, my only method to test multiple version was to introduce a downgraded version :-P